src/web/components/MostViewed/MostViewedFooter/MostViewedFooter.test.tsx

```diff
@@ -5,8 +5,9 @@ import {
     responseWithOneTab,
 } from '@root/fixtures/mostViewed';
 import { useApi as useApi_ } from '@root/src/web/lib/api';
-import { MostViewedFooter } from './MostViewedFooter';
+import { MostViewedFooterData } from './MostViewedFooterData';
```

Comment:
Don't try to test via MostViewedFooter because mocking the IntersectionObserver is tiresome